WordPress/Classes/ViewRelated/Menus/MenuDetailsViewController.m

```diff
@@ -43,15 +44,37 @@ - (void)viewDidLoad
     [self setupTextFieldDesignViews];
     [self setupDoneButton];
     [self setupTrashButton];
+    [self setupObservers];
 
     [self updateTextFieldDesignIconPositioning];
 }
 
+-(void)dealloc
```

Comment:
If I recall correctly, starting with iOS 9, it is not necessary to remove observer on dealloc. I am not saying we should remove this code, just that in theory it is not necessary 😉